src/components/ProgressBar/component.js

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
```

Comment:
there's no functional difference here -- it's only required `React` exists.

however, unless they've changed something...react doesn't actually have a default export. babel adds that itself to libraries and afaik, it's the only reason `import React from 'react'` even works.

`import * as React from React` to me just makes more sense to me because I know for sure what I'm getting back.